src/Providers/ZeroRPCContextProvider.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Juwai\LaravelZeroRPC\Providers;
+
+use Illuminate\Support\ServiceProvider;
+use ZeroRPC\Hook\ConfigMiddleware;
+use ZeroRPC\Context;
+
+class ZeroRPCContextProvider extends ServiceProvider
+{
+    /**
+     * Indicates if loading of the provider is deferred.
+     *
+     * @var bool
+     */
+    protected $defer = true;
+
+    /**
+     * Bootstrap the application services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        $this->publishes([
+            __DIR__ . '/../../config/zerorpc.php' => config_path('zerorpc.php'),
```

Comment:
Would it be possible to use a [Laravel Path Helper](https://laravel.com/docs/5.3/helpers#paths) instead of using `__DIR__` and manually writing the entire path?